bin/scripts/syncWithProject.js

```diff
@@ -0,0 +1,25 @@
+const { TscWatchClient } = require('tsc-watch/client');
+const { execSync } = require('child_process');
+const project = process.argv[2];
+
+const watch = new TscWatchClient();
+
+if (typeof project !== 'string') {
+  throw new Error('Expect string for argument project');
+}
+
+const syncChanges = () => {
+  console.log('Copying changes to', project);
+  execSync('yarn copyAdditionalFiles && yarn buildDeclaredStyles', {
+    stdio: 'inherit',
+  });
+  execSync(
+    `cp -r ./dist/ ../${project}/node_modules/@lifeomic/react-native-sdk`,
```

Comment:
Would be sweet to reuse `copyToProject` here if easy enough

Reply:
`copyToProject` is setup to clear the `dist` folder and run the build command which is handled separately by `tsc-watch`.